content/activities/overview.md

```diff
@@ -0,0 +1,16 @@
+---
+weight: 4
+bookFlatSection: true
+title: "Programs and Activities Sponsored by SIGSOFT"
+aliases: "programs"
+---
+
+ACM SIGSOFT sponsors many activities and events.
+Here is a non-exhaustive list of them.
+
+- [CARES]({{< ref "/cares/SIGSOFT_CARES.md" >}} "cares")
+- [SIGSOFT Travel Support and Childcare Assistance at Conferences - CAPS]({{< ref "CAPSMAIN.md" >}} "caps")
+- [WEBINARS]({{< ref "webinarsmain.md" >}} "webinars")
+- [Summer/Winter Schools]({{< ref "schools.md" >}} "schools")
+- [Student Mentoring Workshop (SMeW)](https://conf.researchr.org/track/icse-2025/icse-2025-smew---student-mentoring-workshop#About)
+  - Africa Initiative. See [here](https://yixue-zhao.medium.com/from-zero-to-one-lessons-learned-on-bringing-african-students-to-international-conferences-74623059c4d4) and [here](https://conf.researchr.org/track/icse-2024/icse-2024-smew---student-mentoring-workshop)
```

Comment:
maybe enrich text a bit: "here for 2025" and "here for 2024"

Reply:
updated! thanks